src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateAllele.java

```diff
@@ -41,7 +41,7 @@
      * All allele list typed as a {@code List<Allele>} as required by some elements in the VCF framework.
      */
     public static final List<Allele> PLAIN_ALL_ALLELES =
-            ALL_ALLELES.stream().map(a -> (Allele) a).collect(Collectors.toList());
+            Collections.unmodifiableList(ALL_ALLELES.stream().map(a -> (Allele) a).collect(Collectors.toList()));
```

Comment:
Don't remember if this came from my branch, but I think at some point I realized that `ALL_ALLELES` is already unmodifiable so it's not necessary to wrap with `Collections.unmodifiableList` here.